domain/src/main/java/org/oppia/android/domain/util/StateRetriever.kt

```diff
@@ -162,7 +163,7 @@ class StateRetriever @Inject constructor(
     answerGroupJson: JSONObject,
     interactionId: String
   ): AnswerGroup {
-    return AnswerGroup.newBuilder()
+    var answerGroup = AnswerGroup.newBuilder()
```

Comment:
A nit: this should be ``val`` since it's not changing references (always prefer ``val`` and only use ``var`` when needed). ``answerGroupBuilder`` would also be clearer and less ambiguous, but I think we can avoid this variable all together--see my comment below.

Reply:
Updated - I ended up not using a val.